DOCS-139: FINAL. EN review, RU translation. (#8391)
ClickHouse/ClickHouse: docs/en/query_language/select.md

#### FINAL Modifier {#select-from-final}

> Applicable when selecting data from tables from the [MergeTree](../operations/table_engines/mergetree.md)-engine family other than `GraphiteMergeTree`. When `FINAL` is specified, ClickHouse fully merges the data before returning the result and thus performs all data transformations that should happen during merges for the given table engine.

Reviewer: ```suggestion
Applicable when selecting data from tables from the [MergeTree](../operations/table_engines/mergetree.md)-engine family other than `GraphiteMergeTree`. When `FINAL` is specified, ClickHouse fully merges the data before returning the result and thus performs all data transformations that happen during merges for the given table engine.
```

Also supported for:
- [Replicated](../operations/table_engines/replication.md) versions of `MergeTree` engines.
- [View](../operations/table_engines/view.md), [Buffer](../operations/table_engines/buffer.md), [Distributed](../operations/table_engines/distributed.md), and [MaterializedView](../operations/table_engines/materializedview.md) engines that operate over other engines, provided they were created over `MergeTree`-engine tables.

> Queries that use `FINAL` are executed more slowly than similar queries that don't, because:

Reviewer: ```suggestion
Queries that use `FINAL` are executed not as fast as similar queries that don't, because:
```
